templates/3d_compute_median_pulse_height.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>Update this with Noah's new readme link about default values for MPH.</p><p> </p><p> </p><p>This section should tell the user that we use it to track the status of the detector, and when it gets otuside of the accepted range, know to perform a detector sweep to bring it back. </p><p> </p><p>This is a quantitative way to know when the instrument has been running long enough to need, or not need, a new sweep</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>